packages/holocron/src/loadModule.node.js

```diff
@@ -153,14 +153,14 @@ const loadModuleFallbackExternals = async (baseUrl, moduleName) => {
 const validateLegacyRequiredExternals = ({
   moduleName,
   holocronModule,
-  providedExternals = {},
+  providedExternals,
```

Comment:
what's the need for removing the empty default?

Reply:
with the added check on 163 its unnecessary and the default is never used.

Reply:
huh, I thought 163 was driven from this, rather that the other way around
why is line 163 changed?

Reply:
Oh! `providedExternals = {}` is not applied when the value to the argument `providedExternals` is something other than `undefined`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#passing_undefined_vs._other_falsy_values
Thanks for the reminder 😅